docs/website/packages/codegen/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+title: Codegen
+---
+
+import DocCardList from '@theme/DocCardList';
+
+[![npm](https://img.shields.io/npm/v/@bonfhir/codegen)](https://www.npmjs.com/package/@bonfhir/codegen)
+
+```bash npm2yarn
+npm install @bonfhir/codegen
+```
+
+The `codegen` package's purpose is the solve the problem of supporting multiple versions of FHIR in the Bonfhir repo. It generate typescript code that will be ready to interact with Fhir, by loading complex the Fhir definitions and integrating them into the code. It simplifies the process of creating the necessary code structure to ensure that the your package or application can correctly handle and use Fhir Resource data. FHIR structure definition files are stored in `packages/codegen/definitions/fhir`.
+
+<DocCardList />
```

Comment:
I would recommend that we add an [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions) here to say that this package exists mainly to support the development of bonFHIR itself. You may use it at your own risk!